ix-dev/stable/nextcloud/templates/test_values/https-values.yaml

```diff
@@ -32,6 +32,9 @@ network:
     proxy_timeout: 60
     use_different_port: true
     external_port: 8443
+    custom_confs:
+      - /foo/bar.conf
+      - /foo/baz.conf
```

Comment:
I don't think we can really test this without creating the actual files.
So lets skip it.